src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -294,8 +294,8 @@ protected function processEntityManager($name, array $defaults)
 		if (empty($config['metadata'])) {
 			$metadataDriver->addSetup('setDefaultDriver', [
 				new Statement($this->metadataDriverClasses[self::ANNOTATION_DRIVER], [
-					[$builder->expand('%appDir%')],
-					2 => $this->prefix('@cache.' . $name . '.metadata')
+					'@' . Doctrine\Common\Annotations\Reader::class,
+					[Nette\DI\Helpers::expand('%appDir%', $builder->parameters)]
```

Comment:
`$builder->expand()` is deprecated and `Nette\DI\Helpers::expand()` is internal. It seems neither is supposed to be used. But it's not clear to me what should be used instead.

Reply:
It can be changed to `$builder->parameters['appDir']` (instead of Helpers::expand), but I have no idea if it's ok.

Reply:
It can be changed to `$builder->parameters['appDir']` (instead of `Helpers::expand`), but I have no idea if it's ok.

Reply:
I'm not sure either. I never used container parameters in extension - at least not directly. I let the user pass them through the extension config.

Reply:
Variant with empty or missing metadata config is feature for quick setup this extension. Any other parameters are "overhead" for users.

IMHO we can use `$builder->expand()` for now (It will be in Nette FW for a [long time](https://forum.nette.org/cs/22093-nette-2-3-0-beta-expan-deprecated)).

Reply:
Agreed.